packages/web-react/src/components/Spinner/__tests__/useSpinnerStyleProps.test.ts

```diff
@@ -0,0 +1,35 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useSpinnerStyleProps } from '../useSpinnerStyleProps';
+
+describe('useSpinnerStyleProps', () => {
+  it('should return defaults', () => {
+    const props = {};
+    const { result } = renderHook(() => useSpinnerStyleProps(props));
+
+    expect(result.current.classProps).toBe('animation-spin-clockwise');
+  });
+
+  it('should return primary color', () => {
```

Comment:
How about making a loop on those colors?

Reply:
Or, the whole text color test could be abstracted and reused across all components using the text color dictionary. But that's surely for another issue 🙂.